staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -254,6 +257,9 @@ type draPlugin struct {
 // The context and/or DRAPlugin.Stop can be used to stop all background activity.
 // Stop also blocks. A logger can be stored in the context to add values or
 // a name to all log entries.
+//
+// If plugin will be used to to publish resources, kubeClient and nodeName
+// parameters are mandatory.
```

Comment:
```suggestion
// If the plugin will be used to to publish resources, [KubeClient] and [NodeName]
// options are mandatory.
```

Reply:
Great, but let's remove the the "to to publish" typo as well.

Reply:
```suggestion
// If the plugin will be used to publish resources, [KubeClient] and [NodeName]
// options are mandatory.
```